openhtf/plugs/README.md

```diff
@@ -0,0 +1,10 @@
+# Usefull thinks!
+## Template to create Plugs
+* (Plug Template)[https://github.com/jo-nas/openhtf_plug_template]: A cookiecutter template for OpenHTF plugs.
```

Comment:
I'm not sure we want to advertise this format for plugs? A single plug per repo might work for some build systems, but it's a pain in others. You're welcome to advertise it in the repos that use it though (like your 4 plugs below)!

Reply:
Agreed we shouldn't advertise/recommend this as the preferred format for plugs. Further, the cookiecutter template also does not appear to contain much if any actual plug code. Aside from a few mentions of the word "plug" in names and docstrings, this template could be used to create almost any new Python package repo. I like the list of plugs below, but let's leave the template out of this doc.